### DISCUSSION TITLE
Can the input stream be overwritten by the output image?

### DISCUSSION THREAD

**Discussion by SanjaySocial** [dlemstra/Magick.NET#1151] [Help]
![image](https://user-images.githubusercontent.com/101013727/156892983-5e3ad6b3-738e-474d-b06a-1eedd798c666.png)
i want to do something like this. Is this fine or am i making some mistake?
Goal is to not end up with 2 copies of the image in memory. @dlemstra pls help

**Comment by dlemstra** [comment]
This might be possible but I all depends on where your stream is coming from. There is nothing in Magick.NET that will make this impossible. You will probably need to do `imageBuffer.SetLength(0);` and maybe `imageBuffer.Position=0` before you write something new to the stream.

**Comment by SanjaySocial** [comment]
it is a memorystream actually

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
